.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Issue
+about: Generic issue template for a proposed OpenAssetIO change
+title: Issue Title
+labels: ''
+assignees: ''
+
+---
+
+## Description
+
+## Acceptance Criteria
+
+- [ ] Requires release note updates.
```

Comment:
[consider] Perhaps we need a reminder like "Requires consideration of dependent projects"?  

Though to be fair, I don't think we've ever forgot that we're about to break a dependent project?  Maybe we shouldn't pre-emptively add reminders.

Reply:
Hmm, i guess I was thinking dependent project breakages would normally be a downstream action, in that we'd see the breakage in the `integrations` check and react, but maybe you're right...

I'll agree that we shouldn't add this pre-emptively for the moment, but it's a good point.